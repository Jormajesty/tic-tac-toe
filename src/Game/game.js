import React, { Component } from "react";
import Board from "../Board/board";

class Game extends Component {
  render() {
    return <div className="Game"> 
      <Board className="boardComp"/>
    </div>;
  }
}

export default Game;
