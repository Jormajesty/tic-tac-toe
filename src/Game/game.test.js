  
import React from "react";
import { shallow, configure } from "enzyme";
import Game from "./Game";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

const game = shallow(<Game />);

describe("when rendering the game", () => {
  it("game renders correctly", () => {
    expect(game).toMatchSnapshot();
  });
  it("Has the Board component", () => {
    expect(game.find(".boardComp").exists()).toBeTruthy();
  });
});