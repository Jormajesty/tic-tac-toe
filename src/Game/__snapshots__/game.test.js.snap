// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when rendering the game game renders correctly 1`] = `ShallowWrapper {}`;
